collection/lens.rkt

```diff
@@ -39,6 +67,21 @@
   (make-lens (λ (target) (nth target index))
              (λ (target value) (set-nth target index value))))
 
+(define (mapper-lens lens)
```

Comment:
I suppose, but `take-lens` and `drop-lens` are decidedly based on verbs. Would you suggest `taker-lens` and `dropper-lens`? :wink: 

Weak arguments-by-example aside, I am of the firm belief that `map-lens` is a better name. I think the name `mapper-lens` implies an additional layer of indirection that doesn't exist. (Say if, for example, it was a lens that somehow used its value as a function for mapping. I don't know.)